ui/scss/component/_button.scss

```diff
@@ -378,13 +383,52 @@ a.button--alt {
   }
 
   &:hover {
+    .button__label {
+      color: var(--color-link);
+    }
+  }
+}
+
+.button--comment-icons {
```

Comment:
I know it's not super D.R.Y. but I would duplicate the button--file-action class contents here.

Then we can get rid of commented out lines.